src/App.js

```diff
@@ -1,10 +1,42 @@
 //TODO: STEP 1 - Import the useState hook.
-import React from "react";
+import React, { useState } from "react";
 import "./App.css";
 import BottomRow from "./BottomRow";
 
 function App() {
-  //TODO: STEP 2 - Establish your applictaion's state with some useState hooks.  You'll need one for the home score and another for the away score.
+  // TODO: STEP 2 - Establish your applictaion's state with some useState hooks.  
+  // You'll need one for the home score and another for the away score.
+
+  const [homeScore, setHomeScore] = useState(0);
+  const [awayScore, setAwayScore] = useState(0);
+
+  //HOME SCORE
+
+  const HomeScoreIncrement = (value) => {
+    setHomeScore(homeScore + value)
```

Comment:
You could have set the value to 7 here since you already know it beforehand, but your implementation is also fien